aws-ts-wordpress-fargate-rds/network.ts

```diff
@@ -0,0 +1,140 @@
+// Copyright 2016-2019, Pulumi Corporation.  All rights reserved.
+
+import { ec2 } from "@pulumi/aws";
+import * as aws from "@pulumi/aws";
+import * as pulumi from "@pulumi/pulumi";
+
+// Interface for VPC args
+export interface VpcArgs {
+  cidrBlock?: string;
+  instanceTenancy?: string;
+  enableDnsHostnames?: boolean;
+  enableDnsSupport?: boolean;
+}
+
+// Creates Vpc
+export class Vpc extends pulumi.ComponentResource {
+  public readonly vpcId: pulumi.Output<string>;
+  public readonly subnetIds: pulumi.Output<string>[] = [];
+  public readonly rdsSecurityGroupIds: pulumi.Output<string>[] = [];
+  public readonly feSecurityGroupIds: pulumi.Output<string>[] = [];
+
+  constructor(name: string, args: VpcArgs, opts?: pulumi.ComponentResourceOptions) {
+
+    super("custom:resource:VPC", name, args, opts);
+
+    const vpcName = `${name}-vpc`;
+
+    const cidrBlock = args.cidrBlock || "10.100.0.0/16";
+    const instanceTenancy = args.instanceTenancy || "default";
+    const enableDnsHostnames = args.enableDnsHostnames || true;
+    const enableDnsSupport = args.enableDnsSupport || true;
+
+    const vpc = new ec2.Vpc(vpcName, {
+      cidrBlock: cidrBlock,
+      instanceTenancy: instanceTenancy,
+      enableDnsHostnames: enableDnsHostnames,
+      enableDnsSupport: enableDnsHostnames,
```

Comment:
update to appropriate variable `enabledDnsSupport`

Reply:
fixed